### ISSUE TITLE
Project Location field should not be shown in a text box

### ISSUE THREAD

**Issue by kflores772** [blackberry/webworks-gui#238]
Even though the text box field for Project Location is non-editable, it is still unusual for it to look like every other editable field in the Configuration page. 

Also it does not make sense for it to be placed with all the editable fields. I suggest placing it somewhere on top before App ID or at the bottom.

2.1.0.13

**Comment by rcruz** [comment]
I agree that the appearance for a non-editable field should be different. A text field can be in a "disabled" state but this appearance differs between browsers. For a more consistent look, css styling can be used instead. The placement could also be better.

Thanks for the feedback :)
